### PR TITLE
feat(appengine): Add AppengineStorageController to list accounts

### PR DESCRIPTION
**Pull request by ewiseblatt**
@danielpeach

### CODE REVIEW

File: clouddriver-appengine/src/main/java/com/netflix/spinnaker/clouddriver/appengine/storage/controllers/AppengineStorageController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/appengine/storage")
```
Comment (lwander) [inline clouddriver-appengine/src/main/java/com/netflix/spinnaker/clouddriver/appengine/storage/controllers/AppengineStorageController.java]: Will this be moved out of the appengine namespace?

**Comment by ewiseblatt** [comment]
Eventually, but not initially. I dont have a handle on it.
There are "DataProviders' in clouddriver. Only S3 is supported. Seems like
it shouldbe related but currently isnt. So keeping this inside appengine
for now.

**Comment by lwander** [comment]
After talking to @ajordens the idea was to replace the data providers with these storage accounts once we're ready

**Comment by ewiseblatt** [comment]
I dont quite understand the data provider so will need to discuss it more
with Tomas. I had an initial exchange with him a couple weeks ago.

**Comment by ewiseblatt** [comment]
FWIW, the issue isnt so much how it works, rather than relating to the
intent and philosophy behind it so I can approach it in some expected way
(or know that this makes sense). Especially to match the intuition of how
usage is expected to evolve.

**Pull request merged**